src/scripting/lua_bases.cc

```diff
@@ -651,7 +651,7 @@ void LuaPlayerBase::__unpersist(lua_State* L) {
 
       (RO) The number of this Player.
 */
-int LuaPlayerBase::get_number(lua_State* L) {
+int LuaPlayerBase::get_number(lua_State* L) {  // NOLINT
```

Comment:
Which check is //NOLINTed here and in the similar lines? Isn't there a better way to fix this?

Reply:
clang-tidy wants to make these functions `const`. However our Luna interface uses function pointers to non-const functions so these functions could no longer be invoked from Lua if their signature was changed.

Reply:
Is it possible to add an explanation to all //NOLINT comments?